sasl/base/src/main/java/org/wildfly/security/sasl/util/AbstractSaslParticipant.java

```diff
@@ -206,6 +206,7 @@ public void setNegotiationState(final int newState) {
      */
     public void negotiationComplete() {
         state = COMPLETE_STATE;
+        log.tracef("SASL Negotiation Succeeded");
```

Comment:
Just another minor comment, but I think would be better to use "SASL Negotiation Completed" here.

Reply:
Just noticed that there are a couple SASL client implementations that call `setNegotiationState(COMPLETE_STATE)` instead of `negotiationComplete()` (see `OAuth2SaslClient` and `ScramSaslClient`). You could update these to call `negotiationComplete()` instead of `setNegotiationState(COMPLETE_STATE)` to make sure your message gets logged for these cases too.

Reply:
Fixed